decidim-admin/spec/system/admin_manages_officializations_spec.rb

```diff
@@ -131,4 +131,49 @@
       end
     end
   end
+
+  describe "contacting the user" do
+    let!(:user) { create(:user, organization: organization) }
+
+    before do
+      click_link "Officializations"
+    end
+
+    it "redirect to conversation path" do
+      within "tr[data-user-id=\"#{user.id}\"]" do
+        click_link "Contact"
+      end
+      expect(page).to have_current_path "/conversations/new?recipient_id=#{user.id}"
+    end
+  end
+
+  describe "clicking on user name" do
+    let!(:user) { create(:user, organization: organization) }
+
+    before do
+      click_link "Officializations"
+    end
+
+    it "redirect to user profile page" do
+      within "tr[data-user-id=\"#{user.id}\"]" do
+        click_link user.name.to_s
+      end
+      expect(page).to have_current_path "/profiles/#{user.nickname}"
```

Comment:
Maybe you can use `decidim.profile_path(user)` here to avoid hardcoding the route.

Reply:
Thanks, didn't know this helper.